sn_node_manager/src/cmd/node.rs

```diff
@@ -492,6 +492,138 @@ pub async fn upgrade(
     Ok(())
 }
 
+/// This functions makes sure that we run exactly the provided count nodes at any time by stopping nodes or
```

Comment:
`This functions makes sure that we run exactly the provided count nodes at any time` =>
`This function ensures running exactly the provided number of nodes at any time`